NAMESPACE

```diff
@@ -27,7 +27,7 @@ export(tstrsplit)
 export(frank)
 export(frankv)
 export(address)
-export(.SD,.N,.I,.GRP,.NGRP,.BY,.EACHI, ., J, patterns, measure)
+export(.SD,.N,.I,.GRP,.NGRP,.BY,.EACHI, ., J)
```

Comment:
please undo this change (patterns and measure should be exported)

Reply:
done 👍